frontend/src/pages/signup/Signup.tsx

```diff
@@ -39,8 +32,19 @@ export default function SignUpPage() {
     try {
       setError("");
       setLoading(true);
-      await signup(email, password);
+      const user = await signup(email, password);
+
+      const config = {
+        headers: {
+          Authorization: `Bearer ${user.user.accessToken}`,
+        },
+      };
+
+      const newUser = { uid: user.user.uid, email: user.user.email };
+
+      await axios.post("/api/user", newUser, config);
```

Comment:
after the changes above, the URL should be just `/user` if I understood correctly